spec/support/shared/multiple_model_examples.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'multiple models' do
+  let(:enum_name) { :status }
+  let(:plural_enum_name) { enum_name.to_s.pluralize }
+  let(:first_model_values) { { active: 'First Active', inactive: 'First Inactive' } }
+  let(:second_model_values) { { pending: 'Second Pending', completed: 'Second Completed' } }
+
+  before do
+    # Define test models
+    stub_const('FirstModel', Class.new(ActiveRecord::Base) do
+      include HumanEnum
+      enum :status, { active: 0, inactive: 1 }
+    end)
+    stub_const('SecondModel', Class.new(ActiveRecord::Base) do
+      include HumanEnum
+      enum :status, { pending: 0, completed: 1 }
+    end)
+
```

Comment:
issue: to keep consistency, please declare these models as normal classes (feel free to wrap them in a module, like `Multiple::`) and, optionally, move them to separate file(s). Maybe I'll move them into a folder inside support, like `support/models`, but that's not a concern for this PR.